app/views/api/messages/_message.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.id message.id
+json.from_user_id message.from_user_id
+json.to_user_id message.to_user_id
+json.title message.title
+json.sent_on message.sent_on.xmlschema
+json.message_read message.message_read
```

Comment:
This allows a sender to see if the recipient has read a message which is not possible at the moment - do we want that?

Reply:
Good catch, thanks.